packages/main/src/MultiComboBox.ts

```diff
@@ -86,305 +92,22 @@ import ResponsivePopoverCommonCss from "./generated/themes/ResponsivePopoverComm
 import ValueStateMessageCss from "./generated/themes/ValueStateMessage.css.js";
 import SuggestionsCss from "./generated/themes/Suggestions.css.js";
 import MultiComboBoxPopover from "./generated/themes/MultiComboBoxPopover.css.js";
+import ComboBoxFilter from "./types/ComboBoxFilter.js";
+import type FormSupportT from "./features/InputElementsFormSupport.js";
+import type ListItemBase from "./ListItemBase.js";
+
+interface IMultiComboBoxItem extends UI5Element {
```

Comment:
do we need this interface to extends UI5Element?

Reply:
Code is dealing with `_id` which comes from the UI5 Element.

Reply:
I'm not sure which is the better solution. To add _id as required class member of classes who implements this interface or to extend UI5Element. If we add it to the interface we will define the minimum scope of working, but if we extend UI5Element we will assume that we can have only custom elements that extends UI5element because implementing the interface will shows error if something from ui5element is missing.